src/cr/cube/matrix.py

```diff
@@ -426,7 +426,7 @@ def columns(self):
                 element,
                 self.table_margin,
                 zscore,
-                np.sum(self._counts, axis=1),
+                **{"opposite_margins": np.sum(self._counts, axis=1)}
```

Comment:
This appears to be a redundant expression. Isn't `opposite_margins=np.sum(self._counts, axis=1),` equivalent? All the `**` says is "take these out of the dict I've put them in". This dict packing can be useful when the contents are unknown (like received from elsewhere as a dict), but in this case they are well defined and defined entirely by this method.